package.json

```diff
@@ -24,7 +24,10 @@
     "yup": "^0.28.3"
   },
   "scripts": {
+    "get-contributors": "bin/generate-contributors-json.js",
+    "prestart": "npm run get-contributors",
```

Comment:
The `pre` scripts are present so that the contributors.json file will be created when `npm run start` or `npm run build` is executed.